### ISSUE TITLE
Bleachbit fails to start

### ISSUE THREAD

**Issue by hidcabl** [bleachbit/bleachbit#501]
Error message: The application has failed to start because its side-by-side configuration is incorrect. Please see the application event log or use the command-line sxstrace.exe tool for more detail.

**Comment by az0** [comment]
Is this when you run the BleachBit installer? Or after installer when running the application?

Is this BleachBit version 2.0, 2.2, or another?

Which version of Windows?

**Comment by az0** [comment]
I am closing for lack of response. If you have the information I need, please reopen this issue or add a comment for me to reopen it.

**Comment by Lochabar** [comment]
I am having the same problem.

It happens when trying to start the application.
It occurred in 2.0 and now in 2.2.
I am using windows 7 home.

![Bleachbit](https://user-images.githubusercontent.com/17562417/56293522-d2679f00-60ee-11e9-9cd7-8f9faeba507e.png)

I have completely uninstalled Bleachbit. Restarted. Reinstalled. And still get the problem.

**Comment by Tobias-B-Besemer** [comment]
Andrew, seems your build environment is out of date...
https://troubleshooter.xyz/wiki/fix-the-application-has-failed-to-start-because-the-side-by-side-configuration-is-incorrect/

**Comment by az0** [comment]
I think the AppVeyor build enviroment is up to date, and it seems to work on most systems. Therefore, it seems to be some idiosyncratic about a few systems.

**Comment by Tobias-B-Besemer** [comment]
The wiki page says this is a Visual Studio 2008 issue... I don't know with that is your Python2EXE compiled...

**Comment by az0** [comment]
@Lochabar  Does installing [The Microsoft Visual C++ 2008 SP1 Redistributable Package (x86)](https://www.microsoft.com/en-us/download/details.aspx?id=5582) fix this error?

Note to self,  Python 2.7 uses MSC v.1500.

For people Googling this error, here it is
>the application has failed to start because its side-by-side configuration is incorrect

**Comment by Lochabar** [comment]
It's already installed. Version 9.0.30729.6161 installed Feb 5.

**Comment by Lochabar** [comment]
Is there any progress with this?

**Comment by az0** [comment]
@Lochabar  Would you please try a recent [CI build](https://ci.bleachbit.org/)? Let me know which version and whether it works. There have been many changes since version 2.2.

**Comment by Lochabar** [comment]
Tried latest release 2.3.1294 from 2019-10-04 - same error "Side-by-side configuration is incorrect"

**Comment by TheAresjej** [comment]
Installing "Microsoft Visual C++ 2008" on Windows 7 fixes the problem. 
(Microsoft Visual C++ 2008 Redistributable - x86 9.0.21022)

But "Visual Studio 2008 reached end of support on April 10, 2018. To aid the discovery of the latest downloads, the links are retained currently, but may be removed in the future." ([source](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads))

**Comment by Lochabar** [comment]
So you are saying I should be using an EARLIER version of "Microsoft Visual C++ 2008" ?
I have 9.0.30729.6161 installed and you want me to use 9.0.21022 instead?

**Comment by TheAresjej** [comment]
No, the developer should update the software so as not to expose its users. 😄
In your case, I would try to reinstall this (2008) version of VSC++ again.

**Comment by Lochabar** [comment]
I uninstalled Microsoft Visual C++ 2008 9.0.30729.6161 and installed the latest 2008 redistributable I could find (turns out to still be 9.0.30729.6161).

I reinstalled BeachBit 2.3.1294. When I started it I got the windows permission window, clicked yes and then nothing else - not the "Side-by-side configuration is incorrect" error and not the BleachBit main window although the process was running in Task Manager.

So I went back to to the latest stable release version 2.2. That installed and works.

So the original error was solved by reinstalling the same version Visual C++ redistributable.
Doesn't explain why a later nightly would not work. And why an old version of Visual C++ is required.

**Comment by az0** [comment]
I would like to revisit this issue after https://github.com/bleachbit/bleachbit/pull/1041 which could have an impact on this
